src/style.css

```diff
@@ -1,3 +1,87 @@
-body {
+* {
```

Comment:
- you can add only the box-sizing inside the `*{}`, other styles you can move to body